_maps/map_files/CTF/cruiser.dmm

```diff
@@ -253,7 +253,7 @@
 /turf/open/floor/iron,
 /area/ctf)
 "mb" = (
-/obj/effect/mob_spawn/human/abductor,
+/obj/effect/mob_spawn/corpse/human/abductor,
```

Comment:
this one line change to a CTF map seems fine 👍

Reply:
kek